atom/common/native_mate_converters/network_converter.cc

```diff
@@ -66,7 +67,9 @@ bool Converter<scoped_refptr<network::ResourceRequestBody>>::FromV8(
     if (type == "rawData") {
       base::Value* bytes = nullptr;
       dict->GetBinary("bytes", &bytes);
-      (*out)->AppendBytes(bytes->GetBlob().data(), bytes->GetBlob().size());
+      (*out)->AppendBytes(
+          reinterpret_cast<const char*>(bytes->GetBlob().data()),
+          base::checked_cast<int>(bytes->GetBlob().size()));
```

Comment:
Could this potentially be
```
(*out)->AppendBytes(bytes->GetBlob());
```
`bytes->GetBlob()` is of type `BlobStorage` which is defined as `std::vector<uint8_t>` here: https://chromium.googlesource.com/chromium/src/+/70.0.3538.109/base/values.h#83, and `AppendBytes` takes a `std::vector<char>`...